fem/fe/fe_base.cpp

```diff
@@ -635,6 +635,115 @@ void ScalarFiniteElement::ScalarLocalL2Restriction(
    }
 }
 
+void NodalFiniteElement::CreateLexicographicFullMap(const IntegrationRule &ir)
```

Comment:
This can be refactored/simplified to significantly reduce duplication, see 5cdfd846357952f33e761ce9056ac154e618d7a0.